tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__Configuration.java

```diff
@@ -19,12 +19,14 @@
  */
 package ${package}.internal;
 
+import org.eclipse.smarthome.config.core.Configuration;
+
 /**
  * The {@link ${bindingIdCamelCase}Configuration} class contains fields mapping thing configuration parameters.
  *
  * @author ${author} - Initial contribution
  */
-public class ${bindingIdCamelCase}Configuration {
+public class ${bindingIdCamelCase}Configuration extends Configuration {
```

Comment:
That's not the idea. Such classes should be merely POJOs, which then get be used in contexts like this:

```java
${bindingIdCamelCase}Configuration config = getThing().getConfiguration().as(${bindingIdCamelCase}Configuration.class);
```

Therefore they must _not_  extend `Configuration`.